miso-web/src/main/webapp/scripts/list.js

```diff
@@ -258,6 +258,9 @@ ListUtils = (function() {
       }
     },
     render : {
+      archived : function(data, type, full) {
+        return data ? "🗄" : "";
```

Comment:
Not sure about this character. I thought it wasn't showing up right until I pasted it into http://www.babelstone.co.uk/Unicode/whatisit.html. In Firefox it does look like it's supposed to - it's just too tiny to tell what it is. On Chrome on my Macbook, it shows up as an empty square. Checkmark might be clearer

Ideally, archived indices just wouldn't be included in the list with current indices